docs/upgrade/UPGRADE-v8.0.0.md

```diff
@@ -119,6 +119,17 @@ There you can find links to upgrade notes for other versions too.
     - we recommend encapsulating collections similarly in your own custom entities as well
 - update your `OrderDataFixture` and `UserDataFixture` to create new users and orders in last two weeks instead of one ([#1147](https://github.com/shopsys/shopsys/pull/1147))
     - this is done by changing all occurrences of `$this->faker->dateTimeBetween('-1 week', 'now');` by `$this->faker->dateTimeBetween('-2 week', 'now');`
+- upgrade `commerceguys/intl` to `^1.0.4` version ([#1192](https://github.com/shopsys/shopsys/pull/1192))
+    - in your `composer.json`, change the dependency:
+        ```diff
+        - "commerceguys/intl": "0.7.4",
+        + "commerceguys/intl": "^1.0.4",
```

Comment:
what about instuctions for monorepo (like run composer update)?

Reply:
:thinking: well, this is a part of general instruction that you should do every time:slightly_smiling_face: https://github.com/shopsys/shopsys/blob/master/docs/contributing/upgrading-monorepo.md#L11